### ISSUE TITLE
SynthText pretrained model

### ISSUE THREAD

**Issue by jianpingZhonggit** [ying09/TextFuseNet.pytorch#70]
Can you provide the pretrained weight on SynthText?
There only offer bouding box annotations of word,character in SynthText? However the TextfuseNet needs segmentation annotation,can you show the code how to get th segmentation annotations? Thank you

**Comment by Real-YeJ** [comment]
@jianpingZhonggit hey, the pretrained-model on SynthText can be download on Google driver. For Synthtext, you can use the bouding box annotations as the segmentation annotations. For your custom datasets, you can use our pretrained-model to generated pseudo labels.

**Comment by jianpingZhonggit** [comment]
Oh, Thany you very much!Using the bouding box as segmentation in character is acceptable.However,can we use the bouding box of the word as segmentation annotation? That means bounding box equals the segmentation

**Comment by jianpingZhonggit** [comment]
@Real-YeJ  Oh, Thany you very much!Using the bouding box as segmentation in character is acceptable.However,can we use the bouding box of the word as segmentation annotation? That means bounding box equals the segmentation.

**Comment by Real-YeJ** [comment]
@jianpingZhonggit  horizontal texts √, multi-oriented and arbitrary-shaped texts ×.

**Comment by jianpingZhonggit** [comment]
@Real-YeJ  Thank you very much! I will try again.
